src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -2004,5 +2020,130 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_Microsoft_PowerShell_Co
                     .GroupByProperty("Label")
                 .EndTable());
         }
+
+        private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Automation_PSStyle()
+        {
+            yield return new FormatViewDefinition("System.Management.Automation.PSStyle",
+                ListControl.Create()
+                    .StartEntry()
+                        .AddItemScriptBlock(@"""$($_.Reset)Example Text""", label: "Reset")
```

Comment:
Instead of using `Example Text` as the output, how about write out the actual ANSI sequences? Just like the formatting of `Get-PSReadLineOption`:

![image](https://user-images.githubusercontent.com/127450/100816046-22959100-33fa-11eb-9018-8b90cfaf4012.png)

Reply:
Good idea.